### PR TITLE
incorporate Hang's suggestions, and also remove all Metadata tables, focus use cases on signaling

### PR DESCRIPTION
**Pull request by danwing**

### CODE REVIEW

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 1]
```diff
@@ -263,13 +268,15 @@ runs over UDP.  As described in {{Section 2.3.7.2 of ?RFC7478}}, there
 is value in differentiating between voice, video and data.  Today's
 video streaming is exclusively over TCP but will migrate to QUIC and
 eventually is likely to support unreliable transport ({{?RFC9221}},
-{{?I-D.kpugin-rush}}).  With unreliable transport of video in
+{{?I-D.draft-ietf-moq-transport}}).  With unreliable transport of video in
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md]:
```suggestion
{{?I-D.ietf-moq-transport}}).  With unreliable transport of video in
```

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 2]
```diff
@@ -445,16 +374,18 @@ There are  cases (crisis) where "normal" network
 resources cannot be used at maximum and, thus, a network would seek to
 reduce or offload some of the traffic during these events -- often
 called 'reactive traffic policy'. An example of such sue case is cellular networks
```
Comment (tireddy2) [inline draft-rwbr-tsvwg-signaling-use-cases.md]: replace "sue" with "such"

**Review by tireddy2 — COMMENTED** [review]
Changes look good to me.

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 3]
```diff
@@ -293,28 +300,37 @@ therefore more critical to deliver to the receiver than delta frames.
...
+Examples: Super bowl, On-Demand Streaming
+
+Requirement:
+
+Signal the flow needs least delay between server and client. Network can provide minimal delay information to the host. Feedback from the network and the client, based on user preference, is required for more efficient streaming. This is required for incorporating special needs based on application/user capabilities and to prioritize traffic during reactive events.
+
+Problems:
+
+1.  All packets prioritized the same irrespective of user preferences/needs:
+    a.  A client based change in priority of a certain type of data is not possible that. Eg. A hearing challenged user can choose video over audio while the priority is different for a regular user.
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md]:
```suggestion
    a.  A client based change in priority of a certain type of data is not possible. For example, a hearing challenged user can choose video over audio while the priority is different for other users.
```

Comment (boucadair) [reply draft-rwbr-tsvwg-signaling-use-cases.md]: to be more inclusive.

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 4]
```diff
@@ -293,28 +300,37 @@ therefore more critical to deliver to the receiver than delta frames.
...
+    b.  Dynamic changes to priority based on user activity is not possible today. Eg. Audio packets having same priority when user mutes the audio, or change in priority during time of emergency where video streaming applications share the same priority as SOS signals.
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md]:
```suggestion
    b.  Dynamic changes to priority based on user activity is not possible today. For example, audio packets having the same priority when a user mutes the audio locally, or change in priority during time of emergency where video streaming applications share the same priority as SOS signals.
```

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 5]
```diff
@@ -293,28 +300,37 @@ therefore more critical to deliver to the receiver than delta frames.
...
+2.  In loss prone network or during a reactive event, retransmissions cause immense delay. Network, not able to distinguish between reliable and loss tolerant data or the critical data within flows(i-frames, p-frames and audio packets), can have challenges in efficiently handling/routing data.
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md]:
```suggestion
2.  In loss-prone networks or during a reactive policy events, retransmissions cause immense delay. Networks, not able to distinguish between reliable and loss-tolerant data or the critical data within a flow (i-frames, p-frames, and audio packets), can have challenges in efficiently handling/forwarding data.
```

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 6]
```diff
@@ -293,28 +300,37 @@ therefore more critical to deliver to the receiver than delta frames.
...
+
+Solution:
 
-Video Streaming Metadata:
+1.  Client to ISP (Host-to-network) signaling can help with conveying user needs.
 
-Based on metadata types listed in the {{?I-D.rwbr-sconepro-flow-metadata}}, the host to network metadata parameters for video streaming type is given below.
+    a. The server (content provider) achieves best scalability by sending a single stream with the same per-packet metadata. The client, on the other hand, signaling the ISP to treat certain packets with a different priority over the others, can help solve the issue. Eg. For video streaming, if the metadata just said "audio" (0x00), "video i-frame" (0x01) "video p-frame" (0x02), the client could have the signaling protocol tell the router which one was most important. Some users could prioritize video over audio and vice versa.
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md]:
```suggestion
    a. The server (content provider) achieves best scalability by sending a single stream with the same per-packet metadata. The client, on the other hand, signaling the ISP to treat certain packets with a different priority over the others, can help solve the issue. For example, for video streaming, if the metadata just said "audio" (0x00), "video i-frame" (0x01) "video p-frame" (0x02), the client could have the signaling protocol tell an upstream router which one was most important. Some users could prioritize video over audio and vice versa.
```

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 7]
```diff
@@ -445,16 +395,18 @@ There are  cases (crisis) where "normal" network
 resources cannot be used at maximum and, thus, a network would seek to
 reduce or offload some of the traffic during these events -- often
 called 'reactive traffic policy'. An example of such sue case is cellular networks
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md]:
```suggestion
called 'reactive traffic policy'. An example of such use case is cellular networks
```

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 8]
```diff
@@ -445,16 +395,18 @@ There are  cases (crisis) where "normal" network
...
-that are overly used (and radio resources exhausted) while alternative network
-attachment networks are available to host.
+that are overly used (and radio resources exhausted) such as a large
+collection of people (e.g., parade, sporting event), or such as a partial
+radio network outage (e.g., tower power outage).  During such an condition, an alternative network
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md]:
```suggestion
radio network outage (e.g., tower power outage).  During such a condition, an alternative network
```

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 9]
```diff
@@ -236,19 +237,23 @@ Metadata exchanges can occur in one single direction or both directions of a flo
 
 The client-centric metadata sharing approach because it preserves privacy and also
 takes advantage of clients having a full view on their available network attachments.
+Without client involvement some use-cases cannot be solved, as detailed below
+in {{generic-cases}}.
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md:241]:
```suggestion
in {{generic-cases}}. 

> Note: some use cases may require seeking for users preference or receiving these preferences. Such matters are
out of scope.
```

**Review by boucadair — CHANGES_REQUESTED** [review]
Some changes from the the airport

**Pull request merged**